CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+### citus-docker v10.1.2.docker (August 18,2021) ###
+
+* Bump Citus version to 10.1.2
+* Bump PG versions to 13.4 and 12.8
```

Comment:
```suggestion

* Bump PG versions to 13.4 and 12.8
```

Sorry for misleading you. Making an edit on the web browser is tricky and I missed an empty line on my previous suggestion.